spec/models/api_azure_search_spec.rb

```diff
@@ -365,35 +363,47 @@
       end
     end
 
-    skip '#as_json' do
+    describe '#as_json' do
```

Comment:
Unskipped this block for code coverage, and moved skips/explanations into specific describe blocks.